### ISSUE TITLE
Chrome Store release

### ISSUE THREAD

**Issue by CHEF-KOCH** [ubeacsec/Silverdog#11]
When the addon gets 'stable' it would be wonderful to release this to the mass within the Chrome Store. 

Speaking of it, is there any open bugtracker/research from the Chromium developers to address this? I'm asking because if it gets migrated into stable builds then this addon seems obsolete. So someone or the developer here should contact or open an ticket (if not already done).

**Comment by mavroudisv** [comment]
Yes there is a plan to release it to the market. We haven't do so already, mainly because as you say the addon is not very stable yet. 

This is a good idea... I will actually open a ticket and see where the discussion goes...
